### PR TITLE
Adding Sushi WMATIC-GHST vault

### PR DESCRIPTION
**Pull request by Qkyrie**
- vault and strategy for wmatic-ghst on sushi
- Aavegotchi GHST token image added. Not sure if this is the correct one.
- Strat: 0x936d1f2ecf2c911efba16bcbb9f205184d827986
- BeefyV6Vault: 0xce069a70925409db10f5df617d947b6b4a4a4202

owner not yet changed to multisig, need verification on the correctness of the contract first. Did full testing of the vault/strat however.

### CODE REVIEW

**Comment by Qkyrie** [comment]
- new contracts deployed
- ownership of vault changed
- ownership of strat changed

**Comment by Qkyrie** [comment]
```
Vault: 0x756866Bd84A2441664e80367977357b1F9b5F3D3
Strategy: 0x42844d785816e53adfFBdFc66d6BFC2E3E83d8FD
Want: 0xf69e93771F11AECd8E554aA165C3Fe7fd811530c
PoolId: 39
```

File: src/features/configure/vault/polygon_pools.js [hunk 1]
```diff
@@ -543,6 +543,31 @@ export const polygonPools = [
...
   },
+  {
+    id: 'sushi-wmatic-ghst',
+    name: 'WMATIC-GHST SLP',
+    token: 'WMATIC-GHST SLP',
+    tokenDescription: 'SushiSwap',
+    tokenAddress: '0xf69e93771F11AECd8E554aA165C3Fe7fd811530c',
+    tokenDecimals: 18,
+    tokenDescriptionUrl: '#',
+    earnedToken: 'mooSushiWMATIC-GHOST',
```
Comment (sircryptomaxi) [inline src/features/configure/vault/polygon_pools.js:554]: GHST not GHOST

File: src/features/configure/vault/polygon_pools.js [hunk 2]
```diff
@@ -543,6 +543,31 @@ export const polygonPools = [
     buyTokenUrl:
       'https://app.sushi.com/swap?inputCurrency=0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619&outputCurrency=0x61BDD9C7d4dF4Bf47A4508c0c8245505F2Af5b7b',
   },
+  {
+    id: 'sushi-wmatic-ghst',
+    name: 'WMATIC-GHST SLP',
```
Comment (sircryptomaxi) [inline src/features/configure/vault/polygon_pools.js:548]: Switch these around so the interesting token is first (and on top in the images), so GHST-MATIC. And drop the W from MATIC in the name, token, and assets.

File: src/features/configure/vault/polygon_pools.js [hunk 3]
```diff
@@ -543,6 +543,31 @@ export const polygonPools = [
...
+    earnContractAddress: '0x756866Bd84A2441664e80367977357b1F9b5F3D3',
+    pricePerFullShare: 1,
+    tvl: 0,
+    oracle: 'lps',
+    oracleId: 'sushi-wmatic-ghst',
+    oraclePrice: 0,
+    depositsPaused: false,
+    status: 'active',
+    platform: 'SushiSwap',
+    assets: ['WMATIC', 'GHST'],
```
Comment (sircryptomaxi) [inline src/features/configure/vault/polygon_pools.js:565]: Switch these around as well, with no W in MATIC

**Review by sircryptomaxi — COMMENTED** [review]
Did you also turn on harvestOnDeposit? If so then you can write the withdrawal fee as 0%.

File: src/features/configure/vault/polygon_pools.js (hunk 2 above)
Comment (Qkyrie) [reply src/features/configure/vault/polygon_pools.js]: > Did you also turn on harvestOnDeposit? If so then you can write the withdrawal fee as 0%.

I did not, wasn't sure this was something I should choose. What's the reasoning behind one or the other?

**Pull request merged**